src/bin/cargo-miri.rs

```diff
@@ -301,17 +303,17 @@ fn in_cargo_miri() {
         Some("test") => (MiriCommand::Test, 3),
         Some("run") => (MiriCommand::Run, 3),
         Some("setup") => (MiriCommand::Setup, 3),
-        // Default command, if there is an option or nothing
+        // Default command, if there is an option or nothing.
         Some(s) if s.starts_with("-") => (MiriCommand::Run, 2),
         None => (MiriCommand::Run, 2),
-        // Unvalid command
+        // Unvalid command.
```

Comment:
"Invalid command" is not a sentence, why should it be ended by a full stop?

Reply:
@RalfJung I agree it's not, but convention seems to be to end in full stops regardless. There's also no choice when you want to follow it by another sentence. It's a debatable one though, I admit!

Reply:
BTW, I just posted this on my cosmetic PR for another repo:

> For me, this is two points: a) sentences should always end in full stops, b) it's still useful to end phrases (not proper sentences) in full stops in case the following line is some commented-out code (that way you know exactly where the comment itself ends), and it's also useful when a full sentence follows the phrase. Finally, consistency with the Rustc codebase (I think?).